crates/markdown_preview/src/markdown_elements.rs

```diff
@@ -221,6 +234,8 @@ pub struct MarkdownHighlightStyle {
     pub strikethrough: bool,
     /// The weight of the text.
     pub weight: FontWeight,
+    /// Whether the text should be stylized as link.
+    pub link: bool,
```

Comment:
@danilo-leal Is this the line you wanted to comment on, right?

Alternative would be to add `color: Hsla` instead of that boolean, and then we can apply the color to highlight style separately, and retain underline `MarkdownHighlight::Style` decoration for links. 

Let me know if this sounds better, or if you have an another idea how to approach that. 🙂